pkg/constant/constant.go

```diff
@@ -86,11 +86,11 @@ const (
 	KubePauseContainerImage            = "registry.k8s.io/pause"
 	KubePauseContainerImageVersion     = "3.9"
 	KubeProxyImage                     = "quay.io/k0sproject/kube-proxy"
-	KubeProxyImageVersion              = "v1.31.5"
+	KubeProxyImageVersion              = "v1.31.6"
 	CoreDNSImage                       = "quay.io/k0sproject/coredns"
 	CoreDNSImageVersion                = "1.11.4"
 	EnvoyProxyImage                    = "quay.io/k0sproject/envoy-distroless"
-	EnvoyProxyImageVersion             = "v1.31.5"
+	EnvoyProxyImageVersion             = "v1.31.6"
```

Comment:
```
"quay.io/k0sproject/envoy-distroless:v1.31.6": quay.io/k0sproject/envoy-distroless:v1.31.6: not found
```
Envoy does not really follow kube versions, it's just a co-incidence that the versions are very close :D